packages/@okta/i18n/src/properties/login.properties

```diff
@@ -699,15 +699,15 @@ cert.authentication.title = Certificate authentication
 ##################################################
 
 ## Password
-oie.password = Password
+oie.password.authenticator.description = Choose a password for your account
```

Comment:
Same is used in verify ?

Reply:
There are no descriptions in verify... So it's only for enroll step